addons/api/addon/models/role.js

```diff
@@ -81,6 +81,32 @@ export default class RoleModel extends GeneratedRoleModel {
     return grantScopes;
   }
 
+  /**
+   * Helper for retrieving grant scope ids that are keywords.
+   * @type {string[]}
+   */
+  get grantScopeKeywords() {
+    return this.grant_scope_ids.filter((id) =>
+      GRANT_SCOPE_KEYWORDS.includes(id),
+    );
+  }
+
+  /**
+   * Helper for retrieving grant_scope_ids that are orgs.
+   * @type {string[]}
+   */
+  get grantScopeOrgIDs() {
+    return this.grant_scope_ids.filter((id) => id.startsWith('o_'));
+  }
+
+  /**
+   * Helper for retrieving grant_scope_ids that are projects.
+   * @type {string[]}
+   */
+  get grantScopeProjectIDs() {
+    return this.grant_scope_ids.filter((id) => id.split('_')[0] === 'p');
```

Comment:
This can get the same `id.startsWith('p_')` as orgs did.

Reply:
updated, thank you!